packages/stellar-axelar-std-derive/src/storage.rs

```diff
@@ -0,0 +1,176 @@
+use heck::ToSnakeCase;
+use proc_macro2::{Ident, TokenStream};
+use quote::{format_ident, quote, ToTokens};
+use syn::{Attribute, Data, DataEnum, DeriveInput, Fields, FieldsNamed, Meta, Type, Variant};
+
+#[derive(Default)]
+struct StorageAttributes {
+    storage_type: StorageType,
+    value_type: Option<Type>,
+}
+
+#[derive(Default, Debug)]
+enum StorageType {
+    #[default]
```

Comment:
A default doesn't really make sense. The type should be mentioned explicitly for clarity